l3kernel/testfiles-plain/m3plain002.tlg

```diff
@@ -0,0 +1,3858 @@
+This is a generated file for the LaTeX (2e + expl3) validation system.
+Don't change this file in any respect.
+{restoring \TYPE=\long macro:#1->\begingroup \long \def \TYPE \ETC.}
+{restoring \l__kernel_expl_bool=\char"1}
+{retaining \__opacity_backend_fill_stroke:ee=\protected\long macro:->\exp_args:Nee \__opacity_backend_fill_stroke:nn }
+{retaining \__pdf_backend_object_now:ne=\protected\long macro:->\exp_args:Nne \__pdf_backend_object_now:nn }
+{retaining \__pdf_backend_object_write:nne=\protected\long macro:->\exp_args:Nnne \__pdf_backend_object_write:nnn }
+{restoring \l_graphics_search_ext_seq=undefined}
```

Comment:
The treatment for `\l_graphics_search_ext_seq` is split into #1469. This makes the m3plain002 output sensitive to whether #1469 is merged.